src/main/java/fr/lunatech/timekeeper/resources/PersonalResource.java

```diff
@@ -51,7 +51,7 @@
     @Counted(name = "countGetPersonalWeek", description = "Counts how many times the user load his personal week on method 'getWeek'")
     @Timed(name = "timeGetPersonalWeek", description = "Times how long it takes the user load his personal week on method 'getWeek'", unit = MetricUnits.MILLISECONDS)
     public WeekResponse getWeek(Integer year, Integer weekNumber) {
-        if(logger.isDebugEnabled()) {
+        if (logger.isDebugEnabled()) {
```

Comment:
just a question, do we really need logger.isDebugEnabled(), in Quarkus is not possible to change log levels in configuration file ...?

Reply:
that's not my code, so I have to invest